.github/workflows/trigger_on_merge_main.yml

```diff
@@ -0,0 +1,15 @@
+# On merge to main
+
+name: Release Automation
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  create-release-and-tag:
+    uses: UserLeap/common-workflows/.github/workflows/_generic_create_release_tag.yml@main
+    with:
+      release_url: https://api.github.com/repos/UserLeap/analytics-react-native-plugin-sprig-test/releases
```

Comment:
for the release_url maybe you wanted `analytics-react-native-plugin-sprig` ?

Reply:
correct